game/resource/addon_english.txt

```diff
@@ -414,5 +419,10 @@
         "Script_6_Closing_2" "We will miss you young one. Make sure to say goodbye before you leave. You are now ready."
         "Script_6_Closing_3" "Don't feel bad if you lose, you're going to lose a lot."
         "Script_6_Closing_4" "That’s true, but Dota is one of the most rewarding games to play if you give it a chance. The map is yours my child. Good luck, and have fun!"
+        "Script_6_anger" "What’s popping gamer?! My name is AngerMania and I make lore videos on Youtube. If you feel overwhelmed with all this new knowledge and you’re worried that you’re not gonna ever get good at this game, you could always take a break from the gameplay and learn about the lore! My main content is videos about specific heroes, so maybe your favourite one has already been covered, or maybe learning the lore of a new hero will inspire you to try them out in the game. Check out the link for my playlist of all my Dota 2 hero lore videos. Good luck learning DOTA, maybe we will meet in game one day. PEACEE"
+        "Script_6_BSJ" "Hey! My name is BananaSlamJamma and I make educational videos for Dota 2. If you want to learn more about some fundamentals that'll help you out when you're getting started check out my new player tutorials by clicking the link. You can also find me on twitch and ask me questions live. Welcome to Dota 2."
+        "Script_6_Purge" "Hey guys this is Purge..."
+        "Script_6_Tsunami" "What's up? I'm Tsunami. If you start following Dota 2 esports you'll probably hear more of me. But once upon a time, I also had no idea what was going on in this crazy game. So I decided to document every time I saw a weird interaction or some spell mechanic that you'd only know after hours of playing a hero. I ended up compiling all of those tips on my site, howdoiplay.com. Take a look if you want some quick advice on how to play or counter specific heroes."
+        "Script_6_DotaU" "Hello! Welcome to DotA 2! I'm Staged and I'm here to represent Dota University. We're a beginner-friendly Discord community that organises coached lobbies in which you face other players with similar experience. We also host educational workshops and tournaments regularly! Check us out if you're looking to learn more about the game or just want to hang out in a fun community! Click on the link below to get an invite to the server! Hope to see you there! (Discord invite link: https://discord.gg/ymSCwmSaTj)"
```

Comment:
Change DotA to Dota.
I assume he doesn't say the link, so maybe make an appropriate panel with the link?